src/components/TaskDetails/TaskDetails.tsx

```diff
@@ -109,7 +111,7 @@ const TaskDetails: React.FC<TaskDetailsProps> = ({ history, showSnackbar }) => {
   }, [updateTaskError, showSnackbar]);
 
   const renderTaskDetails = ({ task }: Query) => (
-    <Box className={wrapper} data-testid='task-details-page'>
```

Comment:
why you removed the task-details-page testId here?

Reply:
@lucianojss It was duplicated, we have the same below in `AppWrapper`:
```  
 <AppWrapper data-testid='task-details-page'>
```